File: backend/api/config/production.py
from .settings import BaseConfig

class ProductionConfig(BaseConfig):
    # Environment
    ENV: str = "production"
    
    # Frontend URL (to be set in environment variables)
    FRONTEND_URL: str = "https://your-domain.com"
    
    # API Settings
    DEBUG: bool = False
    
    # Database (if needed)
    DATABASE_URL: str = "postgresql://user:password@localhost/dbname"
    
    # AI Model Settings
    AI_MODEL: str = "gpt-4o-mini"
    AI_TEMPERATURE: float = 0.0

File: frontend/src/config/production.ts
import type { BaseConfig } from './settings';

export const productionConfig: BaseConfig = {
  API_URL: import.meta.env.VITE_API_URL || 'https://api.your-domain.com',
  PROJECT_NAME: 'Football Betting AI',
  ENV: 'production',
  ENABLE_DEEP_RESEARCH: true,
}; 